source/frontend/src/utils/utils.ts

```diff
@@ -227,3 +228,40 @@ export default function componentLoader(lazyComponent: Promise<any>, attemptsLef
     });
   });
 }
+
+export function capitalizeFirstLetter(value: string) {
+  return value.charAt(0).toUpperCase() + value.slice(1);
+}
+
+export function relationshipTypeToPathName(
+  relationshipType: ApiGen_CodeTypes_DocumentRelationType,
+) {
+  switch (relationshipType) {
+    case ApiGen_CodeTypes_DocumentRelationType.Templates:
+      return 'template';
+
+    case ApiGen_CodeTypes_DocumentRelationType.ResearchFiles:
+      return 'research';
+
+    case ApiGen_CodeTypes_DocumentRelationType.AcquisitionFiles:
+      return 'acquisition';
+
+    case ApiGen_CodeTypes_DocumentRelationType.Leases:
+      return 'lease';
+
+    case ApiGen_CodeTypes_DocumentRelationType.Projects:
+      return 'project';
+
+    case ApiGen_CodeTypes_DocumentRelationType.ManagementActivities:
```

Comment:
this would be the issue for activities, you don't want to navigate to a management file using the id of an activity.

Reply:
right, the updated code fixes this